ROADMAP.md

```diff
@@ -0,0 +1,191 @@
+
+- We distribute versions following an explicit release process (no more `:edge` by default)
+- It's easy to follow changes (CHANGELOG)
+- We distribute binaries
+
+#### Milestone 3: Testground provides extensive documentation
+
+- The documentation is up-to-date
+  - Generates configuration documentation
+- We provide introduction guides for every language
+- We provide doc for Common Patterns
+
+#### Milestone 4: Potential users know about Testground's existence and features
+
+- "Public Relations".
+
+### 4. Testground Development follows high-standard
+
+**Why:** Testground has already provided value amongst many different projects. Now, we need bulletproof development processes to make the project sustainable and facilitate external contributions.
+
+#### Testground CI & Testing is practical and reliable
+
+(I feel like this should be a top-level point)
+
+- We use a single CI (GitHub CI)
+- Refactor the testing to be easier to use and simpler to maintain
+  - remove shell scripts
+- Plan for testing EKS
+- Measure & remove flakiness
+
+#### I have all the documentation I need as a Maintainer
+
+- public specifications
+- public discussions
+- community gatherings
+
+#### Efficient Project Management and Project Visibility
+
+- We have a single maintainer team for review,
+- We have a clear label and triaging process,
+- We have a reliable & transparent contribution process (protected branches, etc),
+- We have precise project management tooling (re-ignite ZenHub?)
+
+### 5. Testground provides the networking tooling required to test complex Distributed / Decentralized Applications
+
+**Why:** This is Testground's main feature.
+
+#### Milestone 1: ExplicitNetwork Simulation Support
+
+There is a clear matrix of what features can be used with which Testground runner.
+
+- MTU issues, networking simulation, etc.
+- This will be a Matrix of feature x runner x precision (transport level, application level, etc.).
+
+#### Milestone 2: Testground provides support for important networks topologies
+
+- Access to public networks - [issue 1472](https://github.com/testground/testground/issues/1472)
+- NAT simulation - [issue 1299](https://github.com/testground/testground/issues/1299)
+- Complex topologies - [issue 1354](https://github.com/testground/testground/issues/1354)
+
+#### Milestone 3: Testground provides a way to run tests over real networks
+
+- Remote runners feature
+
+### 6. Testground provides the tooling to make test maintenance & finding issues simple
+
+**Why:**
+
+- composition files specification and improvements -
+- Logging improvements - [Epic 1355](https://github.com/testground/testground/issues/1355)
+- tcpdump'ing features - [Issue #1384](https://github.com/testground/testground/issues/1384)
+
+### 7. Testground covers every /essential/combination of languages and libraries required by its users
+
+**Why:** Drive adoptions outside of Protocol Labs.
+
+#### We support the primary languages needed by our users
```

Comment:
Should we consider improving the experience of using testground using an "unsupported" language as an alternative?